ckanext/datagovuk/action/create.py

```diff
@@ -80,7 +81,10 @@ def resource_create(context, data_dict):
                 timestamp_str = timestamp.strftime("%Y-%m-%dT%H-%M-%SZ")
```

Comment:
I notice the commit message talks about timestamps, and this variable has timestamp in the name... is this the relevant timestamp?

Reply:
yes, thats the one that was in use by ckanext-s3-resources extension that we grabbed the code from